src/Core/src/Handlers/Image/ImageHandler.Android.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Android.Widget;
+using AndroidX.AppCompat.Widget;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class ImageHandler : ViewHandler<IImage, ImageView>
+	{
+		protected override ImageView CreateNativeView() => new AppCompatImageView(Context);
+
+		public static void MapAspect(ImageHandler handler, IImage image)
+		{
+			handler.NativeView?.UpdateAspect(image);
+		}
+
+		public static void MapIsAnimationPlaying(ImageHandler handler, IImage image)
+		{
+			handler.NativeView?.UpdateIsAnimationPlaying(image);
+		}
+
+		public static async void MapSource(ImageHandler handler, IImage image) =>
+			await MapSourceAsync(handler, image);
+
+		public static async Task MapSourceAsync(ImageHandler handler, IImage image)
+		{
+			if (handler.NativeView == null)
+				return;
+
+			var token = handler._sourceManager.BeginLoad();
```

Comment:
I'm probably missing it but where can this token be cancelled from?

Maybe we should gather these up into the viewmapper base and then on disconnect it can sweep through any active tokens and cancel them?

This might be a useful general concept to have in our base mappers

Reply:
The token can be cancelled by calling the `CompleteLoad` method with a null value. But you are right, we need these to be cancelled by a disconnect method.

Reply:
For now, I hooked up the cancel with the disconnect. But in a separate PR we need to think about how to do this better.